pages/platform/ai/deploy_tuto_10_locust/guide.en-gb.md

```diff
@@ -0,0 +1,252 @@
+
+Select the right tool for the right test is mandatory. For the next parts, **we will select and use locust**, allowing us to show visual graphs. 
+
+
+## Instructions
+
+### Deploy an app with a REST API
+
+Feel free to deploy any app and API that your would like to load test, as long as we can query it via REST queries.
+
+For this tutorial, we will load test a spam classifier API from [AI Deploy app portfolio](https://docs.ovh.com/gb/en/publiccloud/ai/deploy/apps-portfolio/). 
+This API takes as input text sentences (emails), and as an output will provide a spam probability score. 
+
+You can deploy this API easily from our OVHcloud control panel or OVHcloud CLI. A good strategy is to deploy with autoscaling, with a minimum and maximum replicas. This way we will monitor the growth of used replicas.
+
+Here is the CLI command used to deploy it, with autoscaling going from 1 to 5 nodes and a CPU thresold to 75%:
+
+```console
+ovhai app run --name spamclassifier --cpu 1 \
+--auto-min-replicas 1 \
+--auto-max-replicas 5 \
+--auto-resource-type CPU \
+--auto-resource-usage-threshold 75 \
+priv-registry.gra.training.ai.cloud.ovh.net/ai-deploy-portfolio/fastapi-spam-classification
+```
+
+
+### Verify that your API is up and running with cURL
+
+Once deployed, Let's test first our API with a simple cURL command. Here is the command to try in a terminal:
+ 
+```console 
+curl -s -X POST \ 
+"<api_url>/spam_detection_path \
+-H "Authorization: Bearer <token>" \
+-H "Content-Type: application/json" \
+-d '{"message":"This is a test from my machine"}' | jq
+```
+
+Here is the result given by our call:
+
+```console
+{
+  "label": "ham",
+  "label_probability": 0.9875114695528484
+}
+```
+
+Here are a few explanations of the lines:
+ 
+- In the first line, we specify that we will use a post method. 
+- We specify the url where the post request will be executed. The `api_url` is the url of your API. It should look something like this: `https://baac2c13-2e69-4d0f-ae6b-dg9eff9be513.app.gra.ai.cloud.ovh.net/`. 
+- We put the token to access our API, generated via our control panel or `ovhai` CLI. We specify it in the header of the request. 
+- We specify that our body in in a JSON format.
+- We put in our body the message we want to send to the spam classifier. In your case, the body could be different because it depends of the API. And we hope the spam classifier will send us the probability of each response. The last `| jq` instruction allow us to have a good display of the result in the terminal. 
+
+We now have the confirmation that our API is up and running, let's try to load test it. 
+
+We will simply simulate several curl command. With the tool locust, we can simulate several users and define a number of calls to the API we want per minute. This can be easily done with the locust's interface. But before use this interface, we need to launch the locust and configure the tool. This can be easily done with python. Let's do this !
```

Comment:
```suggestion
We will simply simulate several curl command. With the tool locust, we can simulate several users and define a number of calls per minute to the API. This can be easily done with the locust's interface. But before using this interface, we need to launch the locust and configure the tool. This can be easily done with python. Let's do this !
```